contactsummary/js/src/contact-summary/services/contactDetails.js

```diff
@@ -86,6 +86,28 @@ define([
             return deferred.promise;
         }
 
+      /**
+       * Calculate age from birth date
+       *
+       * @param dateOfBirth Date of birth as momentJS acceptable format
+       * @returns string
+         */
+        function calculateAge(dateOfBirth)
+        {
+          dateOfBirth = moment(dateOfBirth, 'YYYY-MM-DD').format('YYYYMMDD');
```

Comment:
This is extremely over engineered , you can achieve the same thing by simply doing

```js
function calculateAge(dateOfBirth) {
  return moment().diff(moment(dateOfBirth, 'YYYY-MM-DD'), 'years');
}